BlockchainSdk/Common/API/MultiNetworkProvider.swift

```diff
@@ -33,7 +34,16 @@ extension MultiNetworkProvider {
                 guard let self = self else { return .anyFail(error: error) }
                 
                 if let moyaError = error as? MoyaError, case let .statusCode(resp) = moyaError {
-                    Log.network("Switchable publisher catched error: \(moyaError). Response message: \(String(describing: String(data: resp.data, encoding: .utf8)))")
+                    let message = "Switchable publisher catched error: \(moyaError). Response message: \(String(describing: String(data: resp.data, encoding: .utf8)))"
+                    
+                    Log.network(message)
+                    
+                    self.exceptionHandler?.errorSwitchApi(
+                        exceptionHost: currentHost,
+                        selectedHost: self.nextHostProvider(for: currentHost),
+                        code: resp.statusCode,
+                        message: message
+                    )
```

Comment:
Предлагаю так назвать параметры и метод

```suggestion
                    self.exceptionHandler?.handleAPISwitch(
                        currentHost: currentHost,
                        nextHost: self.nextHostProvider(for: currentHost),
                        statusCode: resp.statusCode,
                        message: message
                    )
```

Reply:
Или failedHost вместо current

Reply:
> Или failedHost вместо current

Переименовал как в предложенном варианте